packages/schema-blocks/src/functions/presenters/SidebarWarningPresenter.ts

```diff
@@ -64,24 +64,39 @@ export function replaceVariables( issue: analysisIssue ): string {
  */
 function getAnalysisConclusion( validation: BlockValidation, issues: analysisIssue[] ): sidebarWarning {
 	if ( issues.some( issue => issue.result === BlockValidation.MissingBlock ||
-							   issue.result === BlockValidation.MissingAttribute ) ) {
+		issue.result === BlockValidation.MissingAttribute ) ) {
 		return {
 			text: __( "Not all required blocks are completed! No " + issues[ 0 ].parent +
-			" schema will be generated for your page.", "wpseo-schema-blocks" ),
+				" schema will be generated for your page.", "wpseo-schema-blocks" ),
 			color: "red",
 		} as sidebarWarning;
 	}
 
 	if ( validation === BlockValidation.Valid ||
 		issues.every( issue => issue.result !== BlockValidation.MissingAttribute &&
-							   issue.result !== BlockValidation.MissingBlock ) ) {
+			issue.result !== BlockValidation.MissingBlock ) ) {
 		return {
 			text: __( "Good job! All required blocks are completed.", "wpseo-schema-blocks" ),
 			color: "green",
 		} as sidebarWarning;
 	}
 }
 
+/**
+ * Gathers all validation issues recursively and flattens them into one list.
+ *
+ * @param validation The root validation result.
+ *
+ * @return all validation results.
+ */
+function getAllDescendentIssues( validation: BlockValidationResult ): BlockValidationResult[] {
```

Comment:
```suggestion
function getAllChildIssues( validation: BlockValidationResult ): BlockValidationResult[] {
```
descendAnt of child, de laatste is makkelijker typen :-)

Reply:
Bij 'Child' denk ik altijd aan directe kinderen van iets. Ik heb uiteindelijk voor 'descendant' gekozen, sinds dat de lading beter dekt.

Reply:
Fixed in https://github.com/Yoast/javascript/pull/1071/commits/7748eede442f084c87136bfbd3efdd42e4311a60